docs/nx-cloud/private/ami-setup.md

```diff
@@ -0,0 +1,143 @@
+# Setting up a dedicated NxCloud VM
+
+## AWS EC2
+
+1. Login to your AWS Console and select [the top image published here](https://us-east-1.console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=public-images;imageName=nx-cloud;owner=623002322076;sort=desc:imageName)
```

Comment:
will this force people to create the instances in us-east-1 or can you easily switch to a different region? Ideally link is pointed to just the aws console and region is inferred from aws side?

Reply:
ah yes, good spot, didn't realise there was a specific region in the link - I'll update it to be generic